msa-persistence/src/main/java/cz/muni/fi/pa165/monsterslayeragency/entities/Hero.java

```diff
@@ -25,74 +32,7 @@
 	 Commented this one out as hibernate cannot map array
 	 either change it to list and add annotation @ElementCollection (create separate table for it)
 	 or map it according to this: https://vladmihalcea.com/how-to-map-java-and-sql-arrays-with-jpa-and-hibernate/
-	private Skill[] skills;
+	private ESkill[] skills;
```

Comment:
I am not absolutely sure but if I am aware Enums are just classes and they should have same naming conventions so name Skill should be acceptable. It is a little thing and we can discuss this.